package.json

```diff
@@ -42,33 +42,33 @@
     "compression": "^1.7.4",
     "connect-history-api-fallback": "^1.6.0",
     "debug": "^4.1.1",
-    "del": "^4.1.1",
+    "del": "^5.1.0",
     "express": "^4.17.1",
     "html-entities": "^1.3.1",
-    "http-proxy-middleware": "0.19.1",
-    "import-local": "^2.0.0",
-    "internal-ip": "^4.3.0",
+    "http-proxy-middleware": "^1.0.3",
+    "import-local": "^3.0.2",
+    "internal-ip": "^6.0.0",
     "ip": "^1.1.5",
     "is-absolute-url": "^3.0.3",
     "killable": "^1.0.1",
     "loglevel": "^1.6.8",
     "open": "^7.0.3",
-    "p-retry": "^3.0.1",
+    "p-retry": "^4.2.0",
     "portfinder": "^1.0.25",
-    "schema-utils": "^1.0.0",
+    "schema-utils": "^2.6.6",
     "selfsigned": "^1.10.7",
-    "semver": "^6.3.0",
+    "semver": "^7.3.2",
     "serve-index": "^1.9.1",
     "sockjs": "0.3.20",
     "sockjs-client": "1.4.0",
     "spdy": "^4.0.2",
     "strip-ansi": "^3.0.1",
-    "supports-color": "^6.1.0",
+    "supports-color": "^7.1.0",
     "url": "^0.11.0",
     "webpack-dev-middleware": "^3.7.2",
-    "webpack-log": "^2.0.0",
-    "ws": "^6.2.1",
-    "yargs": "12.0.5"
+    "webpack-log": "^3.0.1",
+    "ws": "^7.2.5",
```

Comment:
/cc @Loonride hope we don't breaking anything, can you look?

Reply:
This module supports only Node.js so not breaking changes.

Reply:
based on ws changelog it should be fine